keyboards/zeal60/keymaps/tusing/keymap.c

```diff
@@ -0,0 +1,50 @@
+// Default layout for Zeal60
+#include "zeal60.h"
```

Comment:
You can replace this with `#include QMK_KEYBOARD_H`

Reply:
@Wilba6582 Line 3 can be deleted when @mechmerlin's suggestion is completed.

`#include QMK_KEYBOARD_H` includes `action_layer.h` automatically.

Reply:
fixed